Library/Homebrew/dependency_collector.rb

```diff
@@ -128,7 +128,7 @@ def parse_symbol_spec(spec, tags)
     when :linux         then LinuxRequirement.new(tags)
     when :macos         then MacOSRequirement.new(tags)
     when :maximum_macos then MacOSRequirement.new(tags, comparator: "<=")
-    when :osxfuse       then OsxfuseRequirement.new(tags)
+    when :macfuse       then MacfuseRequirement.new(tags)
```

Comment:
This will break all existing usage of `:osxfuse` (which is already deprecated) so shouldn't be changed.